curp/tests/common/curp_group.rs

```diff
@@ -38,24 +49,45 @@ impl SnapshotAllocator for MemorySnapshotAllocator {
     }
 }
 
+/// Rocks snapshot allocator
+pub(crate) struct RocksSnapshotAllocator;
+
+#[async_trait]
+impl SnapshotAllocator for RocksSnapshotAllocator {
+    async fn allocate_new_snapshot(&self) -> Result<Snapshot, Box<dyn Error>> {
+        let tmp_path = format!("/tmp/snapshot-{}", uuid::Uuid::new_v4());
```

Comment:
Please consider use [tempfile](https://docs.rs/tempfile/latest/tempfile/#) for temp file creation.

Reply:
Fixed